RealFuels/Localization/zh-cn.cfg

```diff
@@ -114,7 +118,7 @@ Localization
         #RF_FuelPump_FuelPump = 燃油泵
         #RF_FuelPump_PumpRate = 泵送速率
 
-        #RF_FuelTank_Default = 默认
+        #RF_FuelTank_Default = Default
```

Comment:
Is this change intentional?

Reply:
Yeah, can't say why. My intuition made me. 😬